### DISCUSSION TITLE
Cron job to regularly backup

### DISCUSSION THREAD

**Discussion by hadynz** [lra/mackup#1971] [Ideas]
Thanks for your work on this tool. Just tried it now and it looks to be doing exactly its billing.

I had a question on what you think is the best way to regularly run backups in the background. I understand this is a request that could fall outside the remit of this tool. Keen to hear if you have this configured somehow or if its a request that other users might find useful too.

**Comment by joshmedeski** [comment]
Can you give us a more specific example? If you use cloud storage for the backup then it's always backed up. I use git and just push when I commit new changes.

**Comment by hadynz** [comment]
I see the work "backup" is overloaded. 

What I would like is if I use a Program X, as I tweak it's config over time, the changes to config are automatically backed up (ie the mackup backup command is run on confit changes or every x min). 

This means i do not have to run any commands manually and I don't have to keep track that I've updated an application config and I need to trigger a backup. 

Hope that clarifies my request.

**Comment by joshmedeski** [comment]
I see. You could run the command on a CRON job but I'd suggest you manually run it so you know what changes occur when the script run.

You shouldn't have to run the script very often, only when you're adding new applications to your workflow that you've never used before. Hope that helps.
